docs/src/gui/qtvcp-vismach.adoc

```diff
@@ -127,123 +138,159 @@ c.ready()
+//FIXME Ascii(STL|OBJ): what's the second arg ?
+Both functions can take one of two named arguments, either a _filename_
+or _raw data_:
+
+[source,python]
+----
+part = AsciiSTL(filename="path/to/file.stl")
+part = AsciiSTL(data="solid part1 facet normal ...")
+part = AsciiOBJ(filename="path/to/file.obj")
+part = AsciiOBJ(data="v 0.123 0.234 0.345 1.0 ...")
+----
+
+The parts will be created in the Vismach space in the _same locations as
+they occupy in the STL or OBJ space_, meaning it may be possible to
+assemble the model in the CAD package.
+
+=== Build from Geometric Primitives
+
+Alternatively parts can be _created inside the model script from a range
+of shape primitives_.
+
+Many shapes are _created at the origin_ and need to be _moved to
+the required location_ after creation.
+
+*`cylinder = CylinderX(x1, r1, x2, r2)`*::
+*`cylinder = CylinderY(y1, r1, y2, r2)`*::
+*`cylinder = CylinderZ(z1, r1, z2, r2)`*::
+  Creates a _(optionally tapered) cylinder on the given axis_ with the
   given radii at the given points on the axis.
-* `sphere = Sphere(x, y, z, r)` +
-  Creates a sphere of radius r at (x,y,z)
-* `triangle = TriangleXY(x1, y1, x2, y2, x3, y3, z1, z2) +
-  triangle = TriangleXZ(x1, z1, x2, z2, x3, z3, y1, y2) +
-  triangle = TriangleYZ(y1, z1, y2, z2, y3, z3, x1, x2)` +
-  Creates a triangular plate between planes defined by the last two
+*`sphere = Sphere(x, y, z, r)`*::
+  Creates a _sphere of radius r at (x,y,z)_.
+*`triangle = TriangleXY(x1, y1, x2, y2, x3, y3, z1, z2)`*::
+*`triangle = TriangleXZ(x1, z1, x2, z2, x3, z3, y1, y2)`*::
+*`triangle = TriangleYZ(y1, z1, y2, z2, y3, z3, x1, x2)`*::
+  Creates a _triangular plate between planes_ defined by the last two
   values parallel to the specified plane, with vertices given by the
   three coordinate pairs.
-* `arc = ArcX(x1, x2, r1, r2, a1, a2)` +
-  Create an arc shape.
-* `box = Box(x1, y1, z1, x2, y2, z2)` +
-  Creates a rectangular prism with opposite corners at the specified
+*`arc = ArcX(x1, x2, r1, r2, a1, a2)`*::
+  Create an _arc shape_.
+*`box = Box(x1, y1, z1, x2, y2, z2)`*::
+  Creates a _rectangular prism with opposite corners_ at the specified
   positions and edges parallel to the XYZ axes.
-* `box = BoxCentered(xw, yw, zw)` +
-  Creates an xw by yw by zw box centred on the origin.
-* `box = BoxCenteredXY(xw, yw, z)` +
-  Creates a box of width xw / yw and height z.
+*`box = BoxCentered(xw, yw, zw)`*::
+  Creates an xw by yw by zw _box centred on the origin_.
+*`box = BoxCenteredXY(xw, yw, z)`*::
+  Creates a _box ground on WY plane_ of width xw / yw and height z.
 
 Composite parts may be created by assembling these primitives either at
 creation time or subsequently:
 
+[source,python]
 ----
 part1 = Collection([Sphere(100,100,100,50), CylinderX(100,40,150,30)])
 part2 = Box(50,40,75,100,75,100)
 part3 = Collection([part2, TriangleXY(10,10,20,10,15,20,100,101)])
 part4 = Collection([part1, part2])
 ----
 
-== Orientating Model Parts
+== Moving Model Parts
```

Comment:
Maybe "Transforming"?
Is a rotation also a movement? 
Maybe ignore this comment.

Reply:
I believe rotation is indeed also a movement, but may be that "Transformation" is a more mathematicaly accurate word ?
OTOH, some may interpret "transformation" as a change in parts shape instead of position, but that can be explicited...
What do others think ?

Reply:
As this is wording related and as a full content centric pass could be done. I leave it for the next one.